lib/r10k/module/forge.rb

```diff
@@ -61,10 +63,9 @@ def expected_version
     @expected_version
   end
 
-
   # @return [R10K::SemVer] The version of the currently installed module
   def current_version
-    @metadata.read
+    #@metadata.read(@path + 'metadata.json') #Why would it not be already read?
```

Comment:
Gotcha. Certainly for the forge class here, metadata is now read on initialization.